lib/bundler/cli.rb

```diff
@@ -444,6 +444,16 @@ def env
       Env.new.write($stdout)
     end
 
+    desc "add GEM [VERSION]", "Add the specified gem to the bottom of Gemfile"
+    method_option "group", :type => :array, :desc => "Specify groups to add the gem in"
+    method_option "source", :type => :string, :desc => "Specify the gem's source"
+    method_option "pre", :type => :boolean, :default => false, :desc => "Check for newer pre-release gems"
+    method_option "append-timestamp", :type => :boolean, :default => false, :desc => "Append timestamp to Gemfile"
```

Comment:
I think we can just make the option `--timestamp` instead of `--append-timestamp`.